_pages/home/spartacus-faq.md

```diff
@@ -3,131 +3,140 @@ title: Spartacus FAQ
+Spartacus is 100% API-driven; in this case, the SAP Commerce Cloud instance is run in a headless fashion.  The Spartacus storefront makes the necessary calls to SAP Commerce Cloud via the RESTful APIs that are part of the Omni-Commerce Connect extensions (link). This decoupling allows you to separate front-end and back-end development, permitting independent update cycles.
 
-### Do I need to have a particular version of SAP Commerce Cloud in order to work with Spartacus? 
+### Do I need to have a particular version of SAP Commerce Cloud in order to work with Spartacus?
 
-SAP Commerce Cloud 1905 is strongly recommended due to required changes to APIs that will be included in that release. Spartacus does partially work with 1811 and 1808 as well as 6.7, but due to API improvements since then and adoption of new CMS components, some features will not work. For example, SmartEdit and extendable checkout only work in 1905. Spartacus won’t work with anything before 6.7 because Spartacus is CMS-based, and the CMS OCC API was first introduced in 6.7. 
+SAP Commerce Cloud 1905 is strongly recommended due to required changes to APIs that will be included in that release. Spartacus does partially work with 1811 and 1808 as well as 6.7, but due to API improvements since then and adoption of new CMS components, some features will not work. For example, SmartEdit and extendable checkout only work in 1905. Spartacus won’t work with anything before 6.7 because Spartacus is CMS-based, and the CMS OCC API was first introduced in 6.7.
 
-### Does Spartacus integrate with any other SAP products? 
+### Does Spartacus integrate with any other SAP products?
 
-Spartacus libraries will be integrated with SAP Cloud Platform Extension Factory and SAP Commerce Cloud Context Driven Services (Q4 2019 expectation), among other products. These differences will be described in the Spartacus documentation and/or by their respective product documentation. 
+Spartacus libraries will be integrated with SAP Cloud Platform Extension Factory and SAP Commerce Cloud Context Driven Services (Q4 2019 expectation), among other products. These differences will be described in the Spartacus documentation and/or by their respective product documentation.
 
-### How does Spartacus compare to the SAP Commerce Cloud Accelerators?  
+### Which browsers does Spartacus support?
+
+Spartacus runs on the following browsers: Chrome, Firefox, Safari, and Edge. Spartacus also runs on any browser that supports the same, modern standards as these browsers. Spartacus does not support IE11 because IE11 is missing certain modern browser features, such as support for CSS variables.
```

Comment:
It's important to note that we use the Angular framework, we could link to or take https://angular.io/guide/browser-support into account. 
Which framework the final build supports is based on the customer application (we only provide library code). The application settings will dictate the supported javascript version,  the CSS build (postcss) and can add certain polyfills to add missing browser features. 

On top of that message, it's important to mention which browser we support with automated tests.

Reply:
btw, we generally refer to "evergreen browsers"